R-package/R/lgb.Dataset.R

```diff
@@ -325,8 +325,12 @@ Dataset <- R6::R6Class(
         num_col <- 0L
 
         # Get numeric data and numeric features
-        c(lgb.call("LGBM_DatasetGetNumData_R", ret = num_row, private$handle),
-          lgb.call("LGBM_DatasetGetNumFeature_R", ret = num_col, private$handle))
+        c(lgb.call(fun_name = "LGBM_DatasetGetNumData_R"
+                   , ret = num_row
+                   , private$handle),
+          lgb.call(fun_name = "LGBM_DatasetGetNumFeature_R"
+                   , ret = num_col
+                   , private$handle))
```

Comment:
```suggestion
        c(
          lgb.call(
            fun_name = "LGBM_DatasetGetNumData_R"
            , ret = num_row
            , private$handle
          ),
          lgb.call(
            fun_name = "LGBM_DatasetGetNumFeature_R"
            , ret = num_col
            , private$handle
          )
        )
```

Sorry, I missed this one during the first review. Could you please change this indentation to match the other code? I know it's a little weird that we have `c(function_call(), function_call())` here 😆